app/models/medical_record.rb

```diff
@@ -1,3 +1,4 @@
 class MedicalRecord < ApplicationRecord
   belongs_to :patient
-end
+  has_one_attached :image
+end
```

Comment:
the last line must be empty
```suggestion
end

```